src/Crucial/Service/Chargify/products-family

```diff
@@ -0,0 +1,83 @@
+<?php namespace Crucial\Service\Chargify;
+
+class ProductFamily extends AbstractEntity
+{
+    /**
+     * List all products for your site
```

Comment:
Please update this comment to reflect that it is for products in a family, not the entire site.